config/routes/api.rb

```diff
@@ -32,15 +32,14 @@
     end
 
     # INFO don't mix scopes and resources in routes
-    resources :teams, only: [:destroy, :index]  do
-      collection do
-        get :last_teammember_teams
-      end
+    resources :teams, only: [:show, :index, :update, :create, :destroy]  do
       resources :api_users, only: [:create, :destroy, :index], module: 'teams'
       resources :groups, only: ['index'], module: 'teams'
       resources :members, except: [:new, :edit], module: 'teams'
       resources :candidates, only:[:index], module: 'teams'
     end
 
+    resources :last_member_teams, only: [:index]
```

Comment:
Not sure what you mean. No, `last_teammember_teams`_ is not in the `team_controller `anymore? I changed that, because @psunix recommended to not make actions with custom names, but to move them to their own endpoints.

Reply:
I see. Okay yeah I get that point

Reply:
yes, own controller is perfect !
but I think we should move this to /api/teams/last_member_teams since it returns a list of teams.
move the controller to Api::Teams::LastTeamMemberTeamsController